src/components/CustomTabPanel.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from '@mui/material'
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  index: number
+  value: number
+}
+
+export const CustomTabPanel = (props: TabPanelProps) => {
```

Comment:
If you don't need props object further in code please do instant destructurization

Reply:
fixed